nucypher/cli/painting/deployment.py

```diff
@@ -106,41 +106,3 @@ def paint_deployer_contract_inspection(emitter, registry, deployer_address) -> N
         emitter.echo(sep, nl=False)
     else:
         emitter.echo(token_contract_info)
-
```

Comment:
This entire file can go away 🪓 . I don't believe any functions in the file are used.

Makes sense because we no longer do/manage deployments from `nucypher`, but rather `nucypher-contracts`.

Reply:
Done